src/Hangfire.SqlServer/SqlServerConnection.cs

```diff
@@ -341,6 +341,20 @@ public override int RemoveTimedOutServers(TimeSpan timeOut)
                 new { timeOutAt = DateTime.UtcNow.Add(timeOut.Negate()) }));
         }
 
+        public override List<string> GetTimedOutServerId(TimeSpan timeOut)
+        {
+            if (timeOut.Duration() != timeOut)
+            {
+                throw new ArgumentException("The `timeOut` value must be positive.", "timeOut");
```

Comment:
use nameof and also this method looks very similar to the one above is there a way you could share it?